common/views/components/OpeningHoursTable/OpeningHoursTable.js

```diff
@@ -0,0 +1,30 @@
+import {spacing, font} from '../../../utils/classnames';
+import type {Place} from '../../../model/opening-hours';
+
+type Props = {|
+  id?: string,
+  place: Place,
+  extraClasses?: string
+|}
+
+const OpeningHoursTable = ({id, place, extraClasses}: Props) => (
+  <table id={id && `${id}-panel-${place.id}`} className={`opening-hours__table ${font({s: 'HNL5'})} ${id ? 'js-tabpanel' : ''} ${extraClasses}`}>
+    <caption className={`opening-hours__caption ${id ? 'js-tabfocus' : ''} ${font({s: 'HNM4'})} ${spacing({s: 2}, {padding: ['top', 'bottom']})}`}>{place.name}</caption>
+    <thead className={`opening-hours__thead ${font({s: 'HNM5'})}`}>
+      <tr className='opening-hours__tr'>
+        <th scope='col' className={`opening-hours__th ${spacing({s: 2}, {padding: ['top', 'bottom']})}`}>Day</th>
+        <th scope='col' className={`opening-hours__th ${spacing({s: 2}, {padding: ['top', 'bottom']})}`}>Times</th>
+      </tr>
+    </thead>
+    <tbody className='opening-hours__tbody'>
+      {place.openingHours.regular.map((day) => (
+        <tr key={day.dayOfWeek} className='opening-hours__tr'>
+          <td className='opening-hours__td'>{day.dayOfWeek}</td>
+          {day.opens ? <td className='opening-hours__td'>{`${day.opens} - ${day.closes}`}</td> : <td className='opening-hours__td'>{day.note}</td>}
```

Comment:
Should this be `${day.opens}—${day.closes}` to match style we've used elsewhere (em-dash and no space)?

Reply:
hmm, not sure this was just a lift and shift, so it's always been like that.  The em-dash style is what we use for dates. This is times, not sure if that makes a difference